astrokat/__main__.py

```diff
@@ -88,11 +88,10 @@ def cli(prog,
         # TODO: more complex usage string in separate function
         usage = "%s [options]" \
                 " --yaml <YAMLfile>" % prog
-        description = \
-            "Sources are specified either as part of an observation profile." \
-            " Track one or more sources for a specified time." \
-            " At least one target must be specified." \
-            " Also note the **required** options."
+        description = "Sources are specified either as part of an observation profile." \
```

Comment:
I read somewhere that using backslashes is to be deprecated sometime in the future, I suggest you handle the long/multiline string differently.

See: https://stackoverflow.com/a/10660443/6165344